Doc/library/collections.rst

```diff
@@ -911,6 +922,16 @@ field names, the method and attribute names start with an underscore.
         >>> Pixel(11, 22, 128, 255, 0)
         Pixel(x=11, y=22, red=128, green=255, blue=0)
 
+.. attribute:: somenamedtuple._fields_defaults
+
+   Dictionary mapping field names to default values.
+
+   .. doctest::
+
+        >>> Account = namedtuple('Account', ['type', 'balance'], defaults=[0])
+        >>> Account._fields_defaults
```

Comment:
I think it is better to demonstrate the purposed effect of having default values:
```
>>> Account('premium')
Account(type='premium', balance=0)
```
`_fields_defaults` is for advanced users, which are able to read the documentation and help carefully. The ability to omit arguments in the constructor is the purpose of using the *defaults* argument.

Reply:
That makes sense.  I've expanded the example